src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -239,6 +239,7 @@ export const CheckoutPayment = ({
       },
     } as QuoteInput,
     validationSchema: getCheckoutDetailsValidationSchema(locale, textKeys),
+    validateOnChange: false,
```

Comment:
It makes a lot more sense with the new solution! How come we do not want to validate on change though?

If a user submits the form without an SSN, the error will show on the SSN field. And when the user has entered a valid SSN the error will still show until they press the submit button again if we do not validate on change?

Reply:
`validateOnChange` causes an error to show up as soon as the user starts typing into the input field 🙄 I just think that's annoying

For the SSN-field we actually submit when the user stops typing so that error will go away automatically :)

Reply:
Ah, yes, I agree that's annoying. It's possible to "configure" Formik so that error messages only show when the field is blurred and dirty, but of course that's out of scope for this! 👌

Reply:
Aha yeah that would make more sense.

For now I thought I'd go with standard web behavior (validate on submit) and then let's see if we need something more fancy ;)